deploy/roles/mesos_master/tasks/main.yml

```diff
@@ -548,3 +548,32 @@
   tags:
     - mesos
     - mesos-master
+
+- name: Create empty log files if they don't exist
+  file:
+    path: "{{ item }}"
+    state: touch
+    owner: root
+    group: root
+    mode: 0640
+  with_items: "{{ log_file_locations }}"
```

Comment:
@stirlingv These folders should always exist, and if they don't we do not want files here instead, so this section can be removed.